### PR TITLE
Add category filter for news post

### LINKED ISSUE #290: Add support for blog tags
1. Add support for defining multiple tags in the FrontMatter metadata of a blog post. The definition should look like this:

`tags: tag1, tag2`

2. Add a route ( `/news/tag/{tag}`) and handler for displaying all blog posts with a tag. This may be a new handler or just a modification to `ContentActions::executeNews`. Use `Post::filter` to filter for posts with a tag (`ContentActions::executeFaq` has a good example of this).

3. Modify the view to indicate it is showing just posts with a tag. 

4. Update the view for an individual blog post to show it's tags.

### PR DESCRIPTION
**Pull request by maximest-pierre**
Fixes #290 

Missing frontend component for filtering and also category inside of the news content.

### CODE REVIEW

File: controller/action/ContentActions.class.php [hunk 1]
```diff
@@ -364,4 +364,32 @@ public static function preparePostListPartial(array $vars): array
...
+  public static function preparePostCategoryFilter(string $category)
+  {
+    Response::enableHttpCache();
+
+    $filter_post = [];
+
+    $posts = array_filter(
+      Post::find(static::VIEW_FOLDER_NEWS, Post::SORT_DATE_DESC),
+      function(Post $post) {
+        return !$post->getDate() || $post->getDate()->format('U') <= date('U');
```
Comment (kauffj) [inline controller/action/ContentActions.class.php]: `return (($post->getCategory() === $category) && (!$post->getDate() || $post->getDate()->format('U') <= date('U')));`

File: controller/Controller.class.php [hunk 2]
```diff
@@ -125,6 +125,7 @@ protected static function getRouterWithRoutes(): \Routing\RouteCollector
     $router->any('/yt2', 'AcquisitionActions::executeYT2');
     $router->get('/verify/{token}', 'AcquisitionActions::executeVerify');
 
+    $router->get('/news/category/{category}', 'ContentActions::preparePostCategoryFilter');
```
Comment (kauffj) [inline controller/Controller.class.php]: execute not prepare

File: controller/action/ContentActions.class.php (hunk 1 above)
Comment (kauffj) [reply controller/action/ContentActions.class.php]: (You'll also need to say `use $category`)

**Comment by maximest-pierre** [comment]
Changes are done.

**Pull request merged**